public/data/ianskie26.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Ian Luciano",
+  "type": "personal",
+  "bio": "Fraud analyst that wants to transition as a Frontend engineer",
+  "avatar": "https://github.com/ianskie26.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ianskie26",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ianluciano_",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ianluciano/",
+      "icon": "linkedin"
+    },
```

Please remove this comma. It should fix the CI errors. 🙂
```suggestion
    }
```